base/OpenGLVisitor.cpp

```diff
@@ -132,6 +120,34 @@ void OpenGLVisitor::visit(Group* g) {
 #endif
             vector<int>::iterator indexIter = (*polyIter)->m_vIndex.begin();
             vector<int>::iterator indexEnd = (*polyIter)->m_vIndex.end();
+            #ifdef HAVE_OPENGLES
+            GLuint vbo;
+            glGenBuffers(1, &vbo);
+            glBindBuffer(GL_ARRAY_BUFFER, vbo);
+            float *verts = (float *)calloc((*polyIter)->m_vIndex.size(), 5 * sizeof(float));
```

Comment:
can this dynamic allocation relocated elswhere (use C++ new is prefered)

Reply:
OK